src/cmd/linuxkit/push_azure.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+)
+
+// Process the run arguments and execute run
+func pushAzure(args []string) {
+	flags := flag.NewFlagSet("azure", flag.ExitOnError)
+	invoked := filepath.Base(os.Args[0])
+	flags.Usage = func() {
+		fmt.Printf("USAGE: %s push azure [options] [name]\n\n", invoked)
```

Comment:
`[name]` seems to indicate that the name is optional. Should also be `path` instead of name, like in the qemu runner. Just noticed the GCP backend has the same, need to fix that